### PR TITLE
Zcash: add setConsensusBranchId() to TansactionBuilder

### PR DESCRIPTION
**Pull request by matthewzipkin-bitgo**
https://github.com/BitGo/bitgo-utxo-lib/pull/89 but rebased on v1.7.1

TICKET BG-24753

### CODE REVIEW

**Review by argjv — COMMENTED** [review]
Update package.json version and LGTM

**Comment by matthewzipkin-bitgo** [comment]
Ok thanks - still making sure it runs correctly as a dep to wallet-platform. Once I'm confident it works I'll come back and merge. Also should we call this version `1.7.1`? My concern is if its required using `^` then some package that isn't ready (doesn't have consensus ID set) will pull it down anyway -- but we already have a `1.8.x` and `1.9.x` I think. So...?

File: test/fixtures/README.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Generate test vectors for new ZCash testnet consensus branch IDs
```
Comment (OttoAllmendinger) [inline test/fixtures/README.md:1]: awesome!

long-term we should have a script for this, but we can tackle that when we overhaul this lib

**Review by OttoAllmendinger — APPROVED** [review]
so this only affects how we are building new zcash txs, not how existing ones are parsed?

**Comment by OttoAllmendinger** [comment]
the fact that this changes the default way zcash transactions are built actually suggests we bump the minor version

Alternatively we could instead release a version of this lib that *supports* the new consensusBranchId in a convenient way without defaulting to it. That way we can decide at the callsite somehow.

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]
Actually I realized that this is for an old version. New development should happen on the master branch.

**Comment by argjv** [comment]
We are releasing 1.7.2 because the latest version in master breaks platform and other places. Fixing utxo-lib is not in the roadmap.

**Pull request merged**